certora/specs/methods/IAccessControlDefaultAdminRules.spec

```diff
@@ -0,0 +1,7 @@
+import "./IERC5313.spec"
+
+methods {
+    defaultAdmin() returns(address) envfree
+    pendingDefaultAdmin() returns(address, uint48) envfree
```

Comment:
You are not including some function 
```
defaultAdminDelay() returns (uint48)
pendingDefaultAdminDelay() returns (address, uint48)
...
beginDefaultAdminTransfer
cancelDefaultAdminTransfer
acceptDefaultAdminTransfer
changeDefaultAdminDelay
rollbackDefaultAdminDelay
```

plus the one for easy instrumentation
```
pendingDefaultAdmin_() returns (address)
pendingDefaultAdminSchedule_() returns (uint48)
pendingDelay_() returns (uint48)
pendingDelaySchedule_() returns (uint48)
delayChangeWait_(uint48 newDelay) returns (uint48)
```

Reply:
I was including them previously, but we decided to remove them [here](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4180#discussion_r1166067453). The conclusion after a call we had is that the blocks method doesn't help in anything unless you specify `envfree` or any other summary.

Reply:
Technically it helps if you ever run the rules on a contract that fail to implement them.

Reply:
How? We only found it helpful for `envfree` methods.

Reply:
How? We only found it helpful for `envfree` methods.

Reply:
And?

Reply:
IMO declaring all the function we use as some advantages
- an exhaustive list of function makes it clear which ones are envfree, and which ones are not, without having to dive into the contract
- if someone wants to reuse the rule on a contract that is missing some functions (like an ERC20 that doesn't have a burn function) it would be possible to just plug our rules on top of their contract and get (partial) coverage
- in case we ever want to change the dispatcher settings, that would reduce the changes needed
- just be explicit about what it is the specs needs to fully run

The drawback I see is the maintenance that it requires. That is not really an issue if we add a function to the contract (it will still be considered when doing `method f`) ... it gets bad if we remove a function (in that case the rules are disabled with no error). Removing functions is not something we really do.

Reply:
Can you wait until I request a review? Please

Reply:
So, overall I agree with keeping them. Initially, they were removed because we didn't see value in having but we also agreed that consistency is better.

I readded them because I also see high value on keeping them. Solving since the suggestion is accepted.